pyproject.toml

```diff
@@ -60,13 +61,19 @@ myst_parser = ">=0.18.1,<1.1.0"
 sphinx = "^5.3.0"
 sphinx_rtd_theme = "^1.1.1"
 
+[tool.pytest.ini_options]
+filterwarnings = [
+    "ignore:.*pkg_resources\\.declare_namespace.*:DeprecationWarning",
```

Comment:
This is a weird sphinx-related side-effect: https://github.com/streamlink/streamlink/pull/5167

I've disabled it because it causes a lot of unrelated noise when running the tests even though it's a problem in sphinx-contrib